apps/testing-javascript/src/pages/index.tsx

```diff
@@ -45,6 +45,8 @@ export const getServerSideProps: GetServerSideProps = async ({req, query}) => {
   const purchasedProductsIds =
     commerceProps.purchases?.map((purchase) => purchase.productId) || []
 
+  console.log({userId: commerceProps.userId, purchasedProductsIds})
```

Comment:
I put up this PR because I wasn't seeing the app recognize my signed in user's purchase. I put this logging in place to figure out why. I wasn't expecting to find `userId` as `undefined`. This is what is getting logged in Vercel function logs after I sign in. 

![CleanShot 2023-12-12 at 11 53 18@2x](https://github.com/skillrecordings/products/assets/694063/15c61f86-34f6-4b68-b829-5b744493ea2d)